packages/react-native-editor/package.json

```diff
@@ -65,7 +65,7 @@
 		"react-native-linear-gradient": "https://raw.githubusercontent.com/wordpress-mobile/react-native-linear-gradient/v2.5.6-wp-2/react-native-linear-gradient-2.5.6-wp-2.tgz",
 		"react-native-modal": "^11.10.0",
 		"react-native-prompt-android": "https://raw.githubusercontent.com/wordpress-mobile/react-native-prompt-android/v1.0.0-wp-2/react-native-prompt-android-1.0.0-wp-2.tgz",
-		"react-native-reanimated": "https://raw.githubusercontent.com/wordpress-mobile/react-native-reanimated/2.2.4-wp-1/react-native-reanimated-2.2.4-wp-1.tgz",
+		"react-native-reanimated": "https://raw.githubusercontent.com/wordpress-mobile/react-native-reanimated/update-version-2.4.1/react-native-reanimated-2.4.1-wp-1.tgz",
```

Comment:
We have to update the reference once https://github.com/wordpress-mobile/react-native-reanimated/pull/13 is merged and a tag is created.